Add instructions for Prometheus in configuring client doc (#18)
rhobs/handbook: content/Products/RHOBS/configuring-clients-for-observatorium.md

          prometheus: k8s
EOF
```

> ### Using a self-managed Prometheus server

Reviewer: Nit: could we use title-case to be consistent with the rest of the document?

This section describes the process of sending metrics collected by a Prometheus server to Observatorium.

#### Background

> In order to remote write metrics from a Prometheus server to Observatorium, the server must be configured to authenticate and make requests to the correct URL using the [`remote_write`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) section of the Prometheus configuration file. Prometheus needs to be configured to use OAuth2 for authenticating the remote write requests to Observatorium. This can be done by setting appropriate fields in [`oauth2`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#oauth2) section under [`remote_write`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).

Reviewer: ```suggestion
In order to remote write metrics from a Prometheus server to Observatorium, the server must be configured to authenticate and make requests to the correct URL using the [`remote_write`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) section of the Prometheus configuration file. Prometheus needs to be configured to use OAuth2 for authenticating the remote write requests to Observatorium. This can be done by setting appropriate fields in the [`oauth2`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#oauth2) section under [`remote_write`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
```

> > **NOTE:** OAuth2 support to Prometheus was added in the version 2.27.0. If using an older version of Prometheus without native OAuth2 support, the remote write traffic needs to go through token-refresher, similar to what is described above for Cluster Monitoring Stack.

Reviewer: Fixing grammar and adding link
```suggestion
> **NOTE:** OAuth2 support was added to Prometheus in version 2.27.0. If using an older version of Prometheus without native OAuth2 support, the remote write traffic needs to go through a proxy such as [token-refresher](https://github.com/observatorium/token-refresher/), similar to what is described above for Cluster Monitoring Stack.
```

#### 1. Modify the Prometheus configuration

> The configuration file for Promethehus must be patched to include a remote-write configuration as shown below. `<tenant>`, `<client_id>`, and `<client_secret>` needs to be replaced with appropriate values.

Reviewer: Typo and grammar
```suggestion
The configuration file for Prometheus must be patched to include a remote-write configuration as shown below. `<tenant>`, `<client_id>`, and `<client_secret>` need to be replaced with appropriate values.
```

Reviewer: Also, the doc already made this mistake earlier, but let's be consistent and choose either `remote-write` or `remote write`

Author: In Prometheus documentation, `remote write` is used so I'll lean towards that.
